app/components/DocumentMeta.js

```diff
@@ -154,6 +167,7 @@ function DocumentMeta({
         </span>
       )}
       &nbsp;{timeSinceNow()}
+      {isTasks && <>&nbsp;•&nbsp;{progressBar}</>}
```

Comment:
Can we add a check for `!isTemplate` here, seems weird to show the progress bar on templates as they'll never be filled out 👍